Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    [SerializeField] private Animator _authorsAnimator;
+
+    private void Start()
+    {
+        Time.timeScale = 1;
+    }
+
+    public void Play()
+    {
+        SceneManager.LoadScene("Level 1");
+    }
+
+    public void Authors()
+    {
+        _authorsAnimator.SetBool("IsClicked", !_authorsAnimator.GetBool("IsClicked"));
+    }
+
+    public void Exit()
+    {
+        Application.Quit();
+    }
```

Comment:
https://trello.com/c/HdOqpxY4/3-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%B8-ui-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9